### PR TITLE
Test for published Jes Labels

### PR DESCRIPTION
**Pull request by ruchim**
A test to identify how labels are being published as a part of workflow-level metadata, task-level metadata and the differences between the two based on label syntax requirements.

### CODE REVIEW

File: src/main/resources/standardTestCases/jes_labels.test [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "calls.wf_hello.hello.backendLabels.label-key-2": "label-value-2"
+  "calls.wf_hello.hello.backendLabels.label-key-1": "label-value-1"
+  "calls.wf_hello.hello.backendLabels.cromwell-workflow-id": "cromwell-<<UUID>>"
+  "calls.wf_hello.hello.backendLabels.cromwell-workflow-name": "wf-hello"
+  "calls.wf_hello.hello.backendLabels.wdl-task-name": "hello"
+  "labels.label-key-1": "label-value-1"
+  "labels.label-key-2": "label-value-2"
+  "labels.cromwell-workflow-id": "cromwell-<<UUID>>"
+  "labels.cromwell-workflow-name": "wf_hello"
+}
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/jes_labels.test]: \n

**Comment by mcovarr** [comment]
🏷  I'm going to label this a success.  :+1:  

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/215/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/jes_labels.test (hunk 1 above)
Comment (ruchim) [reply src/main/resources/standardTestCases/jes_labels.test]: :see_no_evil:

**Comment by danbills** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/215/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**